### DISCUSSION TITLE
project setup

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#189] [Q&A] [answered]
I'm still struggling to see the results of my code. I've set up a codespace, which is where I'm editing. I'll attach the screenshots. I think I've run into an error, which I don't know how to fix. 
<img width="934" alt="Screenshot 2023-07-12 144804" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133051222/8e3ff27c-1f35-4467-9213-6dd3b324da0d">
<img width="954" alt="Screenshot 2023-07-12 142251" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133051222/2635a481-ab0d-4540-8486-05d2b41c6fdd">

**Comment by CJ3315** [comment]
Are you signed into your Github account through visual studio? This might be why you are not seeing updates.

**Reply by Skye404** [reply]
I don't think so... teach me more. Visual Studio?

**Reply by CJ3315** [reply]
Visual Studio is the place where you are editing code in your screenshots. To get that to show in Github and commit changes from VS and have them show up in github you need to allow access to github. 
This article helped me out: https://docs.github.com/en/codespaces/developing-in-codespaces/using-github-codespaces-in-visual-studio-code

**Comment by jakegros** [comment]
Here are some helpful[ Intro videos for Visual Studio Code](https://code.visualstudio.com/docs/getstarted/introvideos) that may help.

**Comment by derekscholes** [comment]
One thing I have found, if you add files to your repo after starting a codespace, you will need to create a new codespace. That is why I received this error. Attempt to create a new codespace and see if that works.

**Reply by Skye404** [reply, marked as answer]
I think we did something different to solve this. I met with Ken and got it resolved. Thanks. I'll try this in the future, if I have the same issue.

**Reply by coopsand** [reply]
@derekscholes This isn't exactly true. If you create a file inside of Visual Studio, you can add it to git. In the terminal type `git add [INSERT FILE HERE]`. That will then tell git that you want to start tracking this file and using it. Then commit and push as normal. Does that make sense?
